backend/models/dtos/__init__.py

```diff
@@ -0,0 +1,33 @@
+from functools import wraps
+from flask import request
+from schematics.exceptions import DataError
+
+
+from backend.exceptions import BadRequest
+
+
+def get_validation_errors(e):
```

Comment:
@Aadesh-Baral  Can you give me a sample of `e` passed to `get_validation_errors`?